libraries/AP_Math/control.cpp

```diff
@@ -407,3 +407,15 @@ float kinematic_limit(Vector3f direction, float max_xy, float max_z_pos, float m
     }
     return fabsf(max_z_neg/direction.z);
 }
+
+// calculate the expo function on the normalised input
+// input must be in the range of -1 to 1
+// expo should be less than 1.0 but limited to be less than 0.95
+float input_expo(float input, float expo)
+{
+    if (expo < 0.95) {
```

Comment:
Whats the reason for 0.95 and not 0.99?

Reply:
I had to stop it somewhere and I thought a reduction in slope of 95% was reasonable.

Reply:
To be fair the 0.95 to 0.99 would probably not be requests as that is a rather extreme expo...